src/main/kotlin/uk/gov/justice/digital/hmpps/hmppselectronicmonitoringcreateanorderapi/models/ContactDetails.kt

```diff
@@ -26,4 +26,9 @@ data class ContactDetails(
   @OneToOne
   @JoinColumn(name = "ORDER_ID", updatable = false, insertable = false)
   private val order: Order? = null,
-)
+) {
+  val isValid: Boolean
+    get() = (
+      contactNumber != null
+      )
+}
```

Comment:
Contact number is required when creating the form, so may need to double check the requirements here